Kubernetes/flannel/l2bridge/cni/config/cni.conf

```diff
@@ -1,45 +1,45 @@
-{
-    "cniVersion":  "0.2.0",
-    "name":  "cbr0",
-    "type":  "flannel",
-    "delegate":  {
-                     "type":  "l2bridge",
-                     "dns":  {
-                                 "Nameservers":  [
-                                                     "11.0.0.10"
-                                                 ],
-                                 "Search":  [
-                                                "svc.cluster.local"
-                                            ]
-                             },
-                     "AdditionalArgs":  [
-                                            {
-                                                "Name":  "EndpointPolicy",
-                                                "Value":  {
-                                                              "Type":  "OutBoundNAT",
-                                                              "ExceptionList":  [
-                                                                                    "192.168.0.0/16",
-                                                                                    "11.0.0.0/8",
-                                                                                    "10.124.24.0/23"
-                                                                                ]
-                                                          }
-                                            },
-                                            {
-                                                "Name":  "EndpointPolicy",
-                                                "Value":  {
-                                                              "Type":  "ROUTE",
-                                                              "DestinationPrefix":  "11.0.0.0/8",
-                                                              "NeedEncap":  true
-                                                          }
-                                            },
-                                            {
-                                                "Name":  "EndpointPolicy",
-                                                "Value":  {
-                                                              "Type":  "ROUTE",
-                                                              "DestinationPrefix":  "10.124.24.33/32",
-                                                              "NeedEncap":  true
-                                                          }
-                                            }
-                                        ]
-                 }
-}
+{
+    "cniVersion":  "0.2.0",
+    "name":  "cbr0",
+    "type":  "flannel",
+    "delegate":  {
+                     "type":  "l2bridge",
+                     "dns":  {
+                                 "Nameservers":  [
+                                                     "10.96.0.10"

```

Comment:
Perhaps we should leave this as "<Kube-DNS Service VIP>" since customers will need to customize this for their own environment or use the default Service VIP we've specified here. Either way, this has implications and needs to be coordinated with Service VIP CIDR and Kube-DNS.

Reply:
I also assume that we're going to add instructions or assume Kube-DNS will be forwarding external DNS resolutions. I believe this will no longer be needed after we fix the secondary DNS server issues.

Reply:
I can confirm that kube-DNS works with external DNS resolutions on Ubuntu 16.04 using a single nameserver (kube-DNS).

Preview documentation includes instructions on how to find the IP of kube-DNS server.